### DISCUSSION TITLE
How to create multiple disks with helm chart

### DISCUSSION THREAD

**Discussion by SomniVertix** [longhorn/longhorn#4392] [Q&A]
I have the case where I want to create multiple disks per node at time of installing longhorn. Currently, I only see `defaultSettings.defaultDataPath` but I would like to create more. I know that you can create more in the longhorn UI, but wanted to see how to do that with the Helm chart

**Comment by derekbit** [comment]
@SomniVertix The multiple default disks in default setting are not available for now.

**Comment by SomniVertix** [comment]
ah, okay. Is there a way to do it after the fact in the yaml? Since the helm chart doesn't support it yet

**Reply by derekbit** [reply]
You can add disks to a node after deploying longhorn by
1. Longhorn UI as you mentioned.
2. Add disks to node.longhorn.io resources (node.spec.disks) directly

Here is an example
```
$ kubectl -n longhorn-system get node.longhorn.io derek-c3-small-x86-02 -o yaml
apiVersion: longhorn.io/v1beta2
kind: Node
metadata:
  creationTimestamp: "2022-08-10T03:05:11Z"
  finalizers:
  - longhorn.io
  generation: 1
  name: derek-c3-small-x86-02
  namespace: longhorn-system
  resourceVersion: "7448"
  uid: 7c1261ad-bb03-4b0a-b7f7-e6eea988bf25
spec:
  allowScheduling: true
  disks:
    default-disk-1e241bd0416b3d27:
      allowScheduling: true
      evictionRequested: false
      path: /var/lib/longhorn/
      storageReserved: 141126150144
      tags: []
  engineManagerCPURequest: 0
  evictionRequested: false
  name: derek-c3-small-x86-02
  replicaManagerCPURequest: 0
  tags: []
```

If you would like to add two disks in the node, you can create a patch file like
```
spec:
  disks:
    disk-1:
      allowScheduling: true
      evictionRequested: false
      path: /mnt/disk1
      storageReserved: 0
      tags: []
    disk-2:
      allowScheduling: true
      evictionRequested: false
      path: /mnt/disk2
      storageReserved: 0
      tags: []
```

Then, apply the patch to the node
```
$ kubectl -n longhorn-system patch node.longhorn.io derek-c3-small-x86-02 --type merge --patch-file ./patch.yaml
```

**Reply by SomniVertix** [reply]
ah, thank you! That worked!
